surfactantplugin-stringextract.py/pyproject.toml

```
@@ -0,0 +1,33 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "surfactantplugin_stringextract.py"
```

Since this plugin is using the `binary2strings` module, we could consider naming it after that. (For the checksec.py plugin, the `.py` is actually part of the name of the module it uses -- I don't think adding `.py` at the end of the names is a practice we need to follow).

```suggestion
name = "surfactantplugin-binary2strings"
```